integrations/aws/utils/resources.py

```diff
@@ -261,4 +261,5 @@ async def resync_cloudcontrol(
                     )
                 else:
                     logger.error(f"Error resyncing {kind} in region {region}, {e}")
-                    raise e
+
+                raise e
```

Comment:
Let's also remove the warning log since it's kind of confusing

```suggestion
            except Exception as e:
                logger.error(
                    f"Error resyncing {kind} in region {region} in account {account_id}, {e}"
                )
                raise e

```